crates/harp/src/fixtures/mod.rs

```diff
@@ -61,19 +52,24 @@ pub static IS_TESTING: bool = cfg!(feature = "testing");
 
 static INIT: Once = Once::new();
 
-pub fn r_test<F: FnOnce()>(f: F) {
-    let guard = unsafe { R_RUNTIME_LOCK.lock() };
+/// Run code accessing the R API in a safe context.
+///
+/// Takes a lock on `R_TEST_LOCK` and ensures R is initialized.
+///
+/// Note: `harp::r_task()` should only be used in Harp tests. Use
+/// `ark::r_task()` in Ark tests so that Ark initialisation also takes place.
+pub fn r_task<F: FnOnce()>(f: F) {
```

Comment:
```suggestion
#[cfg(test)]
pub(crate) fn r_task<F: FnOnce()>(f: F) {
```

If you do this, then `harp::fixtures::r_task()` is only available in harp's unit tests - preventing us from accidentally using it from ark, as we've done in signature-help right now.

Harp doesn't have any integration tests, so I think this is good for now.

(Without `#[cfg(test)]` i get yelled at by RA because `r_task()` isn't used anywhere outside the tests)

Reply:
Sounds good. The other way to fix if we add integration tests for harp is to export with `pub`, which would be needed anyway.

Reply:
Good idea. If we do end up exporting, we could consistently use the namespaced form `crate::r_task()` to ensure we use the correct variant.

Reply:
I replaced everything with `crate::r_task()` in tests so that it's obvious we're using the right symbol.